tests/bluetooth/tester/btp_spec.txt

```diff
@@ -1510,3 +1569,15 @@ Events:
 
 		This event indicates that provisioning link has been closed on
 		given bearer.
+
+	Opcode 0x86 - Network receive Event
+
+		Controller Index:	<controller id>
+		Event parameters:	TTL (1 octet)
+					CTL (1 octet)
+					SRC (2 octets)
+					DST (2 octets)
```

Comment:
The chosen information seems a bit arbitrary to me. So we don't need information on which IVI was used, which NetKey Index was used, etc? I suppose the payload is the payload after decryption with the network key? Don't some tests also require verification of data after app key decryption?

Reply:
Of course, we can extend the event information if it'll be needed. I choose those, to satisfy PTS needs.
````
Please confirm the following network packet was received: Network Header: 
    TTL: [0x00]
    CTL: [0x00]
    SRC: [0x0001]
    DST: [0xFFFC]
    TransportPDU: [0x01020304050607080910111213141516]
````
Yes, the payload here is the payload is after decryption.
I did not saw such tests yet.

Reply:
After decryption with the NetKey or after decryption with NetKey+AppKey?

Reply:
decryption with the netkey